src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationWriter.cs

```diff
@@ -477,7 +477,7 @@ private void WriteStructMethod(StructMapping mapping, string n, string ns, objec
             {
                 if (mapping.TypeDesc.Type != null && typeof(XmlSchemaObject).IsAssignableFrom(mapping.TypeDesc.Type))
                 {
-                    throw new PlatformNotSupportedException(typeof(XmlSchemaObject).ToString());
+                    EscapeName = false;
```

Comment:
This is the current behavior we have for ILGen based serialization. See https://github.com/dotnet/corefx/blob/d6cf18753a35701bfe2d27187ec47db225e1dbae/src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs#L2325.